hearth/lib/hearth/signers.rb

```diff
@@ -9,6 +9,17 @@ def sign(request:, identity:, properties:)
         raise NotImplementedError
       end
 
+      def sign_initial_event_stream_request(request:, identity:, properties:)
+        raise NotImplementedError
```

Comment:
Not related to this PR but I don't like this base class anymore, I may change this ..